plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -758,19 +759,23 @@ get_producers_result chain_api_plugin_impl::get_producers( const get_producers_p
     }
 }
 
-std::string chain_api_plugin_impl::get_agent_public_key(chain::account_name account) const {
+std::string chain_api_plugin_impl::get_agent_public_key(chain::account_name account, chain::symbol symbol) const {
     auto& chaindb = chain_controller_.chaindb();
 
     const cyberway::chaindb::index_request request{N(), N(), N(stake.agent), N(bykey)};
 
-    const auto it = chaindb.lower_bound(request, fc::mutable_variant_object()("token_code", chain::symbol(CORE_SYMBOL).to_symbol_code())("account", account));
+    const auto it = chaindb.lower_bound(request, fc::mutable_variant_object()("token_code", symbol.to_symbol_code())("account", account));
```

Comment:
just notice: in [this PR](https://github.com/GolosChain/cyberway/pull/772), I moved signing_key to the candidate table (N(stake.agent)->N(cand.agent))